### ISSUE TITLE
Discuss about 'shuffle' data

### ISSUE THREAD

**Issue by awaibj** [jaungiers/LSTM-Neural-Network-for-Time-Series-Prediction#16]
@jaungiers 
Hello, I have two questions about 'shuffle' data.
(1) in load_data() function 
np.random.shuffle(train)
whether can be replaced in Keras's fit function, like this
model.fit(
		X_train,
		y_train,
		batch_size=512,
		nb_epoch=epochs,
		verbose=1,
		validation_split=0.05,
		shuffle=True )
(2) Like the stock price data, it's a strict time series data. If we shuffle the dataset, we will destroy the sequence information. I'm not sure whether it will affect the result.

**Comment by mikelam14** [comment]
Shuffle is to shuffle training rows. So within each training sample, the time order is preserved, just the sequence of receiving training sample is not in order.
